recipes/OpenMiChroM/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "OpenMiChroM" %}
+{% set version = "0.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OpenMiChroM-{{ version }}.tar.gz
+  sha256: 835ed034621994bde1522760f4601b82721f9f7ad065c3caf1a9a165d15af2b8
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - CLINAME=OpenMiChroM._cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - h5py
+    - numpy
+    - pandas
+    - python >=3.6
+    - scikit-learn
+    - scipy
+    - six
+    - openmm
+
+test:
+  imports:
+    - OpenMiChroM
+    - tests
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://ndb.rice.edu/Open-MiChroM
+  summary: Open-Michrom lib for chromosome simulations
+  license: MIT
```

Comment:
Can you please add the `license_family` also?

Reply:
the license_family was add

Reply:
about the License File, its already on the git and pip package, I also put it here because I think it is necessary.